project-settings/teams-and-users/export-users-history.md

```diff
@@ -1,27 +1,43 @@
 # Export users history
 
+{% hint style="info" %}
+You need to be on a [Forest Admin Pro plan](https://www.forestadmin.com/pricing/) to have access to this feature.
+{% endhint %}
+
+Some organizations with hundreds of collaborators use Forest Admin every day.
+Administrators of such projects usually need to monitor how users are added to the admin panel, which Permission Level, Role, Tags they are given or Teams they are assign to by other collaborator authorized to do so.
+
+Such monitoring is possible using the users history export.
+You can access to this action in the Users Tab of your Project Settings **(1)**.
+Simply click on the Export dropdown **(2)** and select "Export users history" **(3)**.
+
 <figure><img src="../../.gitbook/assets/image (1).png" alt=""><figcaption></figcaption></figure>
 
-Users' history will allow you to go back and check any changes made on your users inside of your project. The changes we currently track are:&#x20;
+The configuration changes that are tracked on the users of you project are:&#x20;
+
+- Teams (create / name update / delete)
+- Roles (create / name update / delete)
```

Comment:
I had put this only after mentioning user-project's team and role change because it would be harder to justify the name of this export as users history if we clearly state here that the export also contains Teams and Roles. I don't know if it was a legitimate concern tho

Reply:
I am sorry I don’t understand the message and the concern :/